website/client/src/components/tasks/spells.vue

```diff
@@ -105,27 +110,28 @@
   }
 
   .spell {
-    background: #ffffff;
-    border: solid 2px transparent;
-    margin-bottom: 1em;
-    box-shadow: 0 2px 2px 0 rgba(26, 24, 29, 0.16), 0 1px 4px 0 rgba(26, 24, 29, 0.12);
-    border-radius: 1000px;
-    color: #4e4a57;
-    padding-right: 0;
-    padding-left: 0;
-    overflow: hidden;
+    width: 72px;
+    height: 72px;
+    border-radius: 2px;
+    box-shadow: 0 4px 4px 0 rgba(26, 24, 29, 0.16), 0 1px 8px 0 rgba(26, 24, 29, 0.12);
+    border: solid 2px #925cf3;
```

Comment:
for this colors, when possible, could you substitute them with variables found (if they exist) in  for example #ffffff would be $white. In this case you'll have to import the file at the top of the css block with `@import '~@/assets/scss/colors.scss';` like it's down in drawer.vue